spec.md

```diff
@@ -128,6 +134,10 @@ the paths to label objects which can be found underneath the group:
 
 Unlisted groups MAY be labels.
 
+### "label""
```

Comment:
Assuming this will need to be rationalized with `color` below. Worth a placeholder mentioning it?

Reply:
All in the next commits that I'll open once this is merged.

Reply:
:+1: